marine/marine.py

```diff
@@ -154,3 +172,31 @@ def _add_or_get_filter(
 
     def __del__(self):
         self._marine.destroy_marine()
+
+    @classmethod
+    def _expand_macros(
+        cls, fields: List[str], macros: Optional[Dict[str, List[str]]]
+    ) -> List[str]:
+        if not macros:
+            return fields
+
+        expanded_fields = [possible_field for field in fields for possible_field in macros.get(field, [field])]
```

Comment:
I don't think you need to define a variable and then return it, just return the list comprehension :)